packages/editor-ui/src/components/CredentialEdit/CredentialEdit.vue

```diff
@@ -420,6 +420,7 @@ async function beforeClose() {
 	}
 
 	if (!keepEditing) {
+		uiStore.activeCredentialType = null;
```

Comment:
why was this added?

Reply:
This modal is setting `activeCredentialType` [here](https://github.com/n8n-io/n8n/blob/25a7b9897ffa6b0fe572792ff3d9c581cd018cfc/packages/editor-ui/src/components/CredentialEdit/CredentialEdit.vue#L990) but it's never reset once it's closed.